sprint6/zadanie1.js

```diff
@@ -0,0 +1,23 @@
+/*
+Zadanie 1. Utwórz funkcję, zadeklaruj w niej tablicę kilku liczb
+całkowitych, oblicz sumę liczb z tablicy i wypisz do konsoli
+tę sumę.
+*/
+
+function sumNumber() {
```

Comment:
Nazwa musi odzwierciedlać to za co funkcja jest odpowiedzialna. W tym przypadku nazwa funkcji powinna brzmieć mniejwięcej `wypiszSumęLiczb` oczywiście po angielsku